library/src/morpheus-master/sass/_defaults.scss

```diff
@@ -41,6 +41,7 @@ $secondary-color:  			 #F00 !default;
 $alt-colour: 				 #d36f00 !default;
 
 $text-color:				 #212121 !default;
+$text-color-disabled:		 #888 !default;
```

Comment:
Now this is a hard-coded decision to make instead of the programmatic lighten() ... doesn't seem like positive change to me. Seems like it would be fine if you made a new variable though that was based on the current text-color but could be overridden.

Reply:
No problem, I'll play around with a percentage till I find one that's close enough to `#888`